labonneboite/web/templates/base.html

```diff
@@ -220,6 +232,20 @@
     })(document,"script","memoButton");
   </script>
 
+  {# Adblock tracking #}
+  {# https://juliencoquet.com/en/2015/12/29/measuring-the-impact-of-adblocking-with-google-analytics/ #}
+  <script src="{{ url_for('static', filename='js/ads.js') }}" type="text/javascript"></script>
+  <script type="text/javascript">// <![CDATA[
+  jQuery(document).ready(checkAds()); 
+          function checkAds() {
+              if (document.getElementById('adsense') != undefined) {
+                  document.write("<script>dataLayer.push({'event':'adBlock','blocked':'false'});</sc" + "ript>");
+              } else {
+                  document.write("<script>dataLayer.push({'event':'adBlock','blocked':'true'});</sc" + "ript>");
+              }
+          }
+  // ]]></script>
+
```

Comment:
Pourquoi externaliser /js/ads.js et pas ce bloc ? On peut même ce morceau de code dans ads.js ?

Reply:
j'ai suivi ce tuto https://juliencoquet.com/en/2015/12/29/measuring-the-impact-of-adblocking-with-google-analytics/